ipaserver/install/ipa_custodia_regen.py

```diff
@@ -0,0 +1,117 @@
+# Copyright (C) 2017  IPA Project Contributors, see COPYING for license
+"""Regenerate Custodia server.keys and config
+
+KEEP THIS SCRIPT COMPATIBLE WITH FREEIPA 4.5
```

Comment:
This kind of notice usually does not work, do we have a way to check compatibility in some tests ?

Reply:
No, I don't see a way to check compatibility.